runtime/doc/api.txt

```diff
@@ -3055,13 +3067,24 @@ nvim_open_win({buffer}, {enter}, {*config})                  *nvim_open_win()*
     should not be used to specify arbitrary WM screen positions.
 
     Example (Lua): window-relative float >lua
-        vim.api.nvim_open_win(0, false,
-          {relative='win', row=3, col=3, width=12, height=3})
+
+        vim.api.nvim_open_win(0, false, {
+          relative='win',
+          row=3,
+          col=3,
+          width=12,
```

Comment:
vertical space in docs is precious, people will often need to compare things and anticipate following text without jumping around. i don't think the old format was hard to read. and the code itself is usually not the main purpose of the examples, rather it's to show the shape of something.

Reply:
Reverted 91b17a5ee9a5f77845779d2198cf15625a556bb1